config/menu_functions.cfg

```diff
@@ -8,7 +8,7 @@ function install {
   
   check_variables
 
-  read -p "How many nodes do you want to run on the Elrond $ATTENTION ? : " NUMBEROFNODES
+  read -p "How many nodes do you want to run on the MultiversX $ATTENTION ? : " NUMBEROFNODES
```

Comment:
How do we migrate from old to new scripts? Should we only copy the variables.cfg file or there is an option on the old scripts?

Reply:
Existing migration function (will revise if needed)